samples/constitutions/default/actions.js

```diff
@@ -929,6 +943,56 @@ const actions = new Map([
       }
     ),
   ],
+  [
+    "add_security_policy",
+    new Action(
+      function (args) {
+        checkType(args.security_policy_raw, "string", "security_policy_raw");
+        checkType(
+          args.security_policy_digest,
+          "string",
+          "security_policy_digest"
+        );
+      },
+      function (args, proposalId) {
+        const digest = ccf.strToBuf(args.security_policy_digest);
+        const raw = ccf.jsonCompatibleToBuf(args.security_policy_raw);
+
+        const redigested_raw = ccf.bufToStr(
+          ccf.digest("SHA-256", ccf.strToBuf(args.security_policy_raw))
+        );
+        const quoted_digest = ccf.bufToStr(
+          hexStrToBuf(args.security_policy_digest)
+        );
+        if (args.security_policy_raw != "" && redigested_raw != quoted_digest) {
```

Comment:
I seems clearer to have this condition earlier, and to only do the redigest if the string isn't empty?

Reply:
Yeah makes sense